themes/default/layouts/page/pulumi-up-2022.html

```diff
@@ -28,35 +30,321 @@ <h1>The Cloud Awakens</h1>
+                                        <img class="w-1/2 pr-2" src="/images/team/joe-duffy.jpg" alt="Joe Duffy" title="Joe Duffy">
+                                        <img class="w-1/2 pr-2" src="/images/team/luke-hoban.jpg" alt="Luke Hoban" title="Luke Hoban">
+                                    </div>
+
+                                    <div class="w-full lg:w-1/3 lg:h-full flex flex-col items-center lg:items-end justify-start pt-8 lg:py-8">
+                                        <event-session-registration-modal
+                                            modal-title="Save Your Spot"
+                                            button-text="Save Your Seat"
+                                            button-class="btn-pulumiup-card"
+                                            event-sessions="{{ .Params.registration.main | jsonify }}"
+                                        >
+                                        </event-session-registration-modal>
+
+                                        <p class="pulumiup-card-top-date pt-4">
+                                            May 4 | 10:40AM PDT
+                                        </p>
+                                    </div>
+                                </div>
+                            </div>
+
+                            <div class="lg:h-64 p-6">
+                                <p class="my-0 mt-2">
+                                    60 min
+                                </p>
+                                <h5 class="mt-0 mt-4">
+                                    PulumiUP 2022 Keynote: Cloud Migration and Multicloud Solutions Forecast
+                                </h5>
+                                <p>
+                                    Learn about “multi cloud solutions yadda yadda and yadda super insightful forecast with
+                                    thought leadership stuff” with Joe Dufy and Luke Hoban.
+                                </p>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </section>
+
+    <section id="why" class="why-attend text-center container mx-auto py-24 px-6 lg:px-0">
+        <h2>Why Attend</h2>
+        <p>
+            Get ready to be inspired, challenged, and transformed. Learn from experts and success stories about
+            early adopters.
+        </p>
+
+        <div class="flex flex-col lg:flex-row">
+            <div class="w-full lg:w-1/3 px-8 mt-8 lg:mb-0">
+                {{ partial "color-icon.html" (dict "icon" "lightning" "icon_color" "yellow") }}
+                <p class="key-feature-description">
+                    Learn from cloud engineering experts about how they manage and solve frustrations
+                    with handling cloud complexity all the way from cloud native applications to structure
```

Comment:
I'm not quite sure what this sentence is saying (are we learning about managing frustrations around cloud complexity from nothing to cloud native and then about structuring engineering orgs?), so I took a crack at making it make more sense.
```suggestion
                    with handling cloud complexity all the way from cloud native applications to structuring
```